mne/epochs.py

```diff
@@ -763,8 +768,46 @@ def drop_bad_epochs(self):
             disk. To avoid reading epochs form disk multiple times, initialize
             Epochs object with preload=True.
 
+        Parameters
+        ----------
+        reject : dict | None
+            Rejection parameters based on peak-to-peak amplitude.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg'.
+            If reject is None then no rejection is done. Example::
+
+                reject = dict(grad=4000e-13, # T / m (gradiometers)
+                              mag=4e-12, # T (magnetometers)
+                              eeg=40e-6, # uV (EEG channels)
+                              eog=250e-6 # uV (EOG channels)
+                              )
+
+        flat : dict | None
+            Rejection parameters based on flatness of signal.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg', and values
+            are floats that set the minimum acceptable peak-to-peak amplitude.
+            If flat is None then no rejection is done.
+
         """
-        self._get_data_from_disk(out=False)
+        if (reject is not None and self._is_reject is True) or \
```

Comment:
On Wed, Apr 1, 2015 at 10:34 AM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/1900#discussion_r27552350>:
>
> >          """
> > -        self._get_data_from_disk(out=False)
> > +        if (reject is not None and self._is_reject is True) or \
>
> can't you avoid the _is_reject attributes but doing something like
>
> if self.reject is not None and reject is not None:
>
> ?
>
thinking about this a bit, yes we don't need that extra attribute. I
removed it.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/1900/files#r27552350>.
>